packages/cli/templates/vue2/index.html

```diff
@@ -1,12 +1,12 @@
+<!-- DO NOT EDIT THIS FILE. -->
+<!-- This file mimics the production environment, -->
+<!-- but editing it won't have any effect -->
+<!-- on the version deployed in production. -->
```

Comment:
Once we finalize this message, I'll replicate it to other templates. Feel free to suggest edits.

Reply:
maybe we should state why it should not be edited and what effects the change might have ?

Reply:
Something like: 

This file mimics the production environment. A change to this file could result in issues when testing your field plugin in dev-mode.

Reply:
https://github.com/storyblok/field-plugin/pull/222/commits/603bd8882ae346f8f3ebc25d973975a808f29d09

nice, I made a bit of change